app/components/polls/polls-edit/polls-edit.html

```diff
@@ -1,143 +1,195 @@
-<top-bar
-    logo-href="/cms/app/polls/"
-    save-function="saveModel"
-    save-disable-when="model.$dirty().length === 0 && !isNew">
-</top-bar>
+<ng-form name="pollForm">
+  <top-bar
+      logo-href="/cms/app/polls/"
+      save-function="saveModel"
+      save-disable-when="(model.$dirty().length === 0 && !isNew) || !pollForm.$valid">
```

Comment:
Since we're removing our dependence on restmod, we might want to consider using angular's oob form validation flags. In this case, we would use `pollForm.$dirty` instead of `model.$dirty()`.